features/creating-and-editing-a-manual.feature

```diff
@@ -51,6 +51,14 @@ Feature: Creating and editing a manual
     When I edit a manual
     Then the manual's sections won't have changed
 
+  @javascript
+  Scenario: Pasting HTML into a manual
+    When I start creating a new manual
+    * I paste HTML into the manual summary
+    * I paste HTML into the manual body
```

Comment:
```suggestion
    And I paste HTML into the manual summary
    And I paste HTML into the manual body
```

We don't seem to have use of an asterisk convention (tbh I didn't even know you could use asterisks)